scripts/check_compliance.py

```diff
@@ -236,11 +236,17 @@ def parse_codeowners(self, git_root, codeowners):
                         if abs_path.endswith("/"):
                             abs_path = abs_path + "**"
                         elif os.path.isdir(abs_path):
-                            error("Wrong syntax: {}".format(abs_path))
+                            self.case.result = Failure(
+                                "Expected / after directory in CODEOWNERS: "
+                                    + abs_path,
+                                "failure")
```

Comment:
Nit: move to previous line? "pylint" is useful for this if you manage to ignore the unrelated noise.

Reply:
Think it's clearer that it's two arguments like this. [This guy](https://www.youtube.com/watch?v=wf-BqAjZb8M) is my spirit animal when it comes to "PEP 8-ifying" code and the like. :)